docs/openstack/create-and-connect-to-the-VM/launch-a-VM.md

```diff
@@ -93,67 +98,79 @@ you may choose any available images.
     is available as a Volume that is attach to the instance to enable persistent
     storage.
 
-- **Flavor** tab: Specify the size of the instance to launch. Choose `cpu-su.4`
-from the 'Flavor' tab by clicking on the "+" icon.
+### **Flavor** Tab
+
+Specify the size of the instance to launch. Choose `cpu-su.4` from the 'Flavor'
+tab by clicking on the "+" icon.
 
 !!! question "Important Note"
     In NERC OpenStack, flavors define the compute, memory, and storage
     capacity of nova computing instances. In other words, a flavor is an
     available hardware configuration for a server.
 
-    Some of the flavors will not be available for your use as per your resource **Quota**
-    limits and will be shown as below:
+    Some of the flavors will not be available for your use as per your resource
+    **Quota** limits and will be shown as below:
 
     ![Flavor Not Avaliable due to Your Quota](images/flavor-not-available-due-to-quota.png)
```

Comment:
it does not display the image.

Reply:
It does not load at all on my screen, it also shows up blue as if it is tabbed in and tabbed in items I don't believe are shown.

Reply:
now I see it is a display issue of comparing Markdown to the NERC html display. looks good.